### PR TITLE
Companion for sub/7040

### PR DESCRIPTION
**Pull request by kianenigma**

### CODE REVIEW

File: runtime/kusama/src/lib.rs [hunk 1]
```diff
@@ -935,6 +938,13 @@ impl frame_support::traits::OnRuntimeUpgrade for CustomOnRuntimeUpgrade {
 	}
 }
 
+pub struct PhragmenElectionDepositRuntimeUpgrade;
+impl frame_support::traits::OnRuntimeUpgrade for PhragmenElectionDepositRuntimeUpgrade {
+	fn on_runtime_upgrade() -> frame_support::weights::Weight {
+		pallet_elections_phragmen::migrations::migrate_to_recorded_deposit::<Runtime>(5 * DOLLARS)
```
Comment (kianenigma) [inline runtime/kusama/src/lib.rs:944]:
```suggestion
		pallet_elections_phragmen::migrations::migrate_to_recorded_deposit::<Runtime>(5 * CENTS)
```

Err this must actually be the old voting bond.. please double check and apply upon approve.

File: runtime/kusama/src/lib.rs [hunk 2]
```diff
@@ -454,13 +454,15 @@ impl pallet_collective::Trait<CouncilCollective> for Runtime {
 
 parameter_types! {
 	pub const CandidacyBond: Balance = 1 * DOLLARS;
-	pub const VotingBond: Balance = 5 * CENTS;
+	pub const VotingBondBase: Balance = 5 * DOLLARS;
```
Comment (kianenigma) [inline runtime/kusama/src/lib.rs:457]: I think a 100x increase here is too much? I think `1 DOLLAR` is a good middle ground.

**Review by thiolliere — DISMISSED** [review]

**Comment by kianenigma** [comment]
So I rested the weight to all zero, hoping that the bot would replace the, realising now that it can't.

File: runtime/kusama/src/lib.rs [hunk 3]
```diff
@@ -889,13 +893,20 @@ impl pallet_proxy::Config for Runtime {
...
+impl frame_support::traits::OnRuntimeUpgrade for SchedulerRuntimeUpgrade {
 	fn on_runtime_upgrade() -> frame_support::weights::Weight {
 		0
 	}
 }
 
+pub struct PhragmenElectionDepositRuntimeUpgrade;
+impl frame_support::traits::OnRuntimeUpgrade for PhragmenElectionDepositRuntimeUpgrade {
+	fn on_runtime_upgrade() -> frame_support::weights::Weight {
+		pallet_elections_phragmen::migrations_3_0_0::apply::<Runtime>(5 * CENTS, DOLLARS)
```
Comment (thiolliere) [inline runtime/kusama/src/lib.rs:906]: I think this won't compile, you have to implement V2ToV3 trait manually

**Review by gavofyork — APPROVED** [review]

**Pull request merged**